examples/send-and-receive-emails/send-and-receive-emails.description

```diff
@@ -0,0 +1,2 @@
+// The Email Connector is used to send (with SMTP) and receive (with POP3 or IMAP4) emails using the SSL or STARTTLS protocols.
+// This sample includes sending and receiving emails over SSL with client objects.
```

Comment:
No need to emphasize the use of client objects.

Reply:
Corrected.